runtime-common/stdlib/string/from-json-processor.h

```diff
@@ -91,13 +90,13 @@ class FromJsonVisitor {
   }
 
   void do_set(JsonRawString &value, const mixed &json) noexcept {
-    kphp_runtime_context.static_SB.clean();
-    if (!impl_::JsonEncoder{0, false, get_json_obj_magic_key()}.encode(json)) {
+    RuntimeContext::get().static_SB.clean();
```

Comment:
Please, minimise the number of static `get` method calls. It's never inlined. It's ok to make just one call and save a reference to a specific context in a variable

Reply:
+